docs/msf/managing-store-configuration.mdx

```diff
@@ -41,36 +47,51 @@ Accept: application/json
 }
 ```
 
+</Tab>
+</Tabs>
+
 Because the request did not specify any query parameters, the API returned the **global default** configuration for this merchant. Responses that return global configuration data will contain values for all properties of the requested object.
 
 The merchant's global configuration is used for all storefronts and other channels that have not specified different, channel-specific configuration settings.
 
 To check whether channel-specific values for a setting are defined, send the `channel_id` as a query parameter in your requests to settings endpoints.
 
-```http title="Example request: Get store profile settings for channel 122"
+<Tabs items={['Request', 'Response']}>
+<Tab>
+
+```http filename="Example request: Get store profile settings for channel 122"
 GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/settings/profile?channel_id=122
 X-Auth-Token: {{ACCESS_TOKEN}}
 Accept: application/json
 ```
-&nbsp;
-```json title="Example response: Get store profile settings for channel 122" lineNumbers
+</Tab>
+<Tab>
+
+```json filename="Example response: Get store profile settings for channel 122" showLineNumbers
 {
   "data": {},
   "meta": {}
 }
 ```
+</Tab>
+</Tabs>
 
 The response's empty `data` object indicates that no channel-specific store profile settings are configured for the specified channel. It is safe to infer that this channel is using the global default store profile settings. When the global default store profile settings change, this channel's profile will reflect those changes. For example, if the global default value for the store name changed, the store name would change at each place it appears in this channel.
 
 The following request-response pair shows an example of a channel that does have channel-specific store profile settings configured.
 
-```http title="Example request: Get store profile settings for channel 123"
+<Tabs items={['Request', 'Response']}>
+<Tab>
+
+```http filename="Example request: Get store profile settings for channel 123"
```

Comment:
```suggestion
```http filename="Example request: Get store profile settings for channel 123" copy
```